src/Miso/Lens.hs

```diff
@@ -324,15 +324,22 @@ infixr 4 <>~
 -- | Execute a monadic action in @MonadState@ that returns a field. Sets the
 -- return value equal to the field in the record.
 --
+-- As a reasonable mnemonic, this lets you store the result of a monadic action in a 'Lens' rather than
+-- in a local variable.
+--
+-- @
+-- do foo <- bar
+--    ...
 -- @
--- newtype List = List { _values :: [Int] }
 --
--- values :: Lens List [Int]
--- values = lens _values $ \\l vs -> l { _values = vs }
+-- will store the result in a variable, while
 --
--- addElement :: List -> List
--- addElement list = list & values <>~ [2]
```

Comment:
This was copy pasted example from <>~.
It's not as easy to come up with a self-contained example for `<~`, so I just copied its explanation from the lens package.